### PR TITLE
chore(charts): refactor helm values for the hostpath pv

### PR DESCRIPTION
**Pull request by tianweidut**
## Description
- remove the outdated `agentHostPathRoot` description
- provide `devMode.createConsoleHostPathPV` fields to config console debug volume
- provide `controller.job.cacheDirHostPath` field to describe job pod cache volume

## Modules
- [x] Others

## Checklist
- [x] run code format and lint check
- [ ] add unit test
- [ ] add necessary doc

### CODE REVIEW

**Review by anda-ren — APPROVED** [review]
LGTM

File: docker/charts/templates/controller-deployment.yaml [hunk 1]
```diff
@@ -12,11 +12,13 @@ spec:
     metadata:
       labels: {{ include "chart.controller.labels" . | nindent 8}}
     spec:
+      {{- if .Values.devMode.createConsoleHostPathPV.enabled }}
       volumes:
-        - name: controller-storage
+        - name: console-hostpath-volume
           hostPath:
-            path: {{ .Values.storage.agentHostPathRoot }}/controller
+            path: {{ .Values.devMode.createConsoleHostPathPV.rootPath }}/controller/console
```
Comment (jialeicui) [inline docker/charts/templates/controller-deployment.yaml:19]: Can we remove the word 'PV' from the field name as it seems to be only a 'host path' and not a 'PV'?

Comment (tianweidut) [reply docker/charts/templates/controller-deployment.yaml:19]: hostPath is a typed volume of PersistentVolume ?

Comment (tianweidut) [reply docker/charts/templates/controller-deployment.yaml]: updated

**Review by jialeicui — APPROVED** [review]
LGTM

**Pull request merged**